lighthouse-plugin-publisher-ads/typings/lh-externs.d.ts

```diff
@@ -17,21 +17,28 @@
  *
  * The export list can be semi-auto generated by running the following command:
  *
- *   find node_modules/lighthouse/types/**.d.ts | sed -E "s#node_modules/(.*).d.ts#import '\1';#g"
+ *   find node_modules/lighthouse/types node_modules/lighthouse/types/lhr -name '*.d.ts' -maxdepth 1 | sed -E "s#node_modules/(.*).d.ts#import '\1';#g"
```

Comment:
someone better with `find` than me probably has a better way to do this :)

there's some types in here that aren't needed for import (jest, smokehouse, enquirer, etc), but they aren't doing much harm so it seems easier to keep the find command simple

Reply:
not wildly better but the `find` bit can be abbreviated a tad with bracket expansion (works in all shells)

`find node_modules/lighthouse/types{,/lhr} -name '*.d.ts' -maxdepth 1`

weird syntax, especially for directory paths, but .. it works.